EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFPackerOutput.h

```diff
@@ -0,0 +1,20 @@
+#include "FWCore/Framework/interface/Event.h"
+
+#include "EventFilter/L1TRawToDigi/interface/Packer.h"
+#include "EventFilter/L1TRawToDigi/plugins/PackerFactory.h"
+
+#include "L1Trigger/L1TMuon/interface/RegionalMuonRawDigiTranslator.h"
+#include "BMTFTokens.h"
+
+namespace l1t {
+   namespace stage2 {
+      class BMTFPackerOutput : public Packer 
+      {
+         public:
+            Blocks pack(const edm::Event&, const PackerTokens*) override;
+         private:
+            std::map<unsigned int, std::vector<uint32_t> > payloadMap_;
```

Comment:
in the end there is one value in the map and it seems like it is just used as a local variable. Is this member data (let alone a map) needed?

Reply:
I see that is the case.  Looks like only the key 123 (BMTF BlockID is 123) is used.

I missed this one, can simplify it in the next PR.